snowpack/src/build/build-import-proxy.ts

```diff
@@ -209,19 +209,20 @@ export async function wrapImportProxy({
   hmr: boolean;
   config: SnowpackConfig;
 }) {
-  const {baseExt, expandedExt} = getExt(url);
 
-  if (typeof code === 'string') {
-    if (baseExt === '.json') {
-      return generateJsonImportProxy({code, hmr, config});
-    }
-
-    if (baseExt === '.css') {
-      // if proxying a CSS file, remove its source map (the path no longer applies)
-      const sanitized = code.replace(/\/\*#\s*sourceMappingURL=[^/]+\//gm, '');
-      return expandedExt.endsWith('.module.css')
-        ? generateCssModuleImportProxy({url, code: sanitized, hmr, config})
-        : generateCssImportProxy({code: sanitized, hmr, config});
+  for (const ext of getExt(url)) {
```

Comment:
😄 You’ve actually picked up on the intended behavior of `getExt()`! We had planned on making this change when we designed the plugin API, but just hadn’t gotten around to it (@FredKSchott and I had designed it after my own personal struggles of dealing with webpack resolvers and fighting to get an expanded extension to match certain loaders and not others).

Glad to see you made good use of something that was already in the codebase.